pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/DefaultCastTrackSelector.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.cast
+
+import androidx.media3.common.TrackSelectionOverride
+import androidx.media3.common.TrackSelectionParameters
+import androidx.media3.common.Tracks
+
+/**
+ * Default cast track selector
+ * Support only [TrackSelectionOverride] from [TrackSelectionParameters.overrides].
+ */
+class DefaultCastTrackSelector : CastTrackSelector {
+
+    override fun getActiveMediaTracks(
+        parameters: TrackSelectionParameters,
+        tracks: Tracks
+    ): LongArray {
+        val trackIdToSelect = mutableListOf<Long>()
+        for (trackGroup in parameters.overrides.keys) {
+            runCatching {
+                trackIdToSelect.add(trackGroup.id.toLong())
+            }
+        }
+        return trackIdToSelect.toLongArray()
```

Comment:
I guess the `runCatching` is there is case the `toLong()` fails? Since we don't do anything with the result of `runCatching`, why not simply do:
```suggestion
        return parameters.overrides.keys
            .mapNotNull { it.id.toLongOrNull() }
            .toLongArray()
```

Reply:
Sounds good yes